transform/StorageTransform.cs

```diff
@@ -108,5 +108,17 @@ public Task<AssetMigrationResult> RunAsync(AssetDetails details, CancellationTok
                 await uploader.UpdateOutputStatus(containerName, cancellationToken);
             }
         }
+
+        public Headers GetHeaders(string filename)
+        {
+            var contentType = Path.GetExtension(filename) switch
+            {
+                ".m3u8" => "application/vnd.apple.mpegurl",
+                ".mpd" => "application/dash+xml",
+                ".vtt" => "text/vtt",
+                _ => "video/mp4"
```

Comment:
For the asset that is generated by MES job, it may contain a list of .json files which are copied over to the destination folder, it might be better to choose a matched mimetype for .json files, instead of using video/mp4.

Reply:
The same might apply to the thumbnail file.

Reply:
Added application/json for json files and "video/mp4" for mp4 files. Rest is left as null so storage can pick default.